packages/ingest-mongodb-public/src/sources/mongodbDotCom/WebDataSource.ts

```diff
@@ -51,7 +51,7 @@ export function makeWebDataSource({
         logger.error(`Failed to fetch pages for source: ${name}`, err);
         throw err;
       } finally {
-        await browserPage.close();
+        await browserPage.close?.();
```

Comment:
idk why i needed to add this to have the tests pass. 

this call was added in a recent PR #676 but tests were passing there 🤷